.buildkite/core.rayci.yml

```diff
@@ -1,6 +1,7 @@
 group: core tests
 steps:
   - label: ":ray: core: python tests"
+    if: build.env("RAY_CI_PYTHON_AFFECTED") != "0" || build.env("RAY_CI_DASHBOARD_AFFECTED") != "0"
```

Comment:
change `RAY_CI_` prefix to `RAYCI_` ? all other ci env vars on civ2 uses `RAYCI_`.

also those env vars are not set yet.

Reply:
Sound good, they are set in this PR: https://github.com/ray-project/rayci/pull/103/files